specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,6389 @@
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          },
+          {
+            "name": "hostPoolUpdate",
+            "description": "Object containing HostPool update definitions.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/HostPoolUpdate"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Successfully initiated hostpool update."
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/controlUpdate": {
+      "post": {
+        "tags": [
+          "HostPoolUpdate"
+        ],
+        "operationId": "HostPools_ControlUpdate",
+        "description": "Control update of a hostpool.",
+        "x-ms-examples": {
+          "HostPools_ControlUpdate_Post": {
+            "$ref": "./examples/HostPools_ControlUpdate_Post.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          },
+          {
+            "name": "hostPoolControlParameter",
+            "description": "Object containing control action for hostpool update.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/HostPoolControlParameter"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Successfully controlled hostpool update."
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/updateStatuses/current": {
```

Comment:
I dont think this is the right name for this endpoint as it does more than just return the update status for e.g. user might be interested in the shc/ update params and will use the same endpoint @datr93

Reply:
how about updateDetails?